pandas/tests/indexes/test_base.py

```diff
@@ -936,6 +934,14 @@ def test_symmetric_difference(self):
         assert tm.equalContents(result, expected)
         assert result.name == 'new_name'
 
+    def test_symmetric_difference_on_equal_multiindex(self):
+        # GH13490
+        idx1 = MultiIndex.from_tuples(self.tuples)
+        idx2 = MultiIndex.from_tuples(self.tuples)
+        result = idx1.symmetric_difference(idx2)
+        expected = MultiIndex(levels=[[], []], labels=[[], []])
+        assert tm.equalContents(result, expected)
```

Comment:
why?

``tm.assert_index_equal(result, expected)`` will work this compares the indexes, including all meta-data.

in fact your example is NOT propogating meta-data (that's what _shallow_copy does and why you need to use it). add some ``names`` for the levels and see.